### PR TITLE
Issue#14

### PR DESCRIPTION
**Pull request by rachelbroders**
Organized files in "pages" folder into component subfolders. Converted any files in class format into component format.
Was also working on fixing an issue with running create/add/delete computer/script when computer/script not selected. Ended up leaving this issue in to be dealt with in a seperate branch.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -5,4 +5,5 @@ REMOTE_FOLDER: "/tmp/Purpl3_RMS/"
 BLACKLIST_CMD_FILE: "res/blacklistCMD.txt"
 DB_PATH: 'purpl3_rms.db'
 OUTPUT_TIMEOUT: 3
-PUBLIC_SSH_KEY_VALUE: 
+PUBLIC_SSH_KEY_VALUE: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQxKZrQbXNYqNK3e2iCvQcAxRBgM7P8Cg9m0z8iDjCAhNmr6HaKX4k3TsDAtHG1VqkKZ5Lo4BoU1g1cbi6RLdKKr/l6wYQ/+h8wN5wpKgW59ZJWW8v8xlT8mpYs/5zwikg+7DMDFThVno5ttvoMYPXhX+/Y0AHL5xaaXoHswbP5ZQq2pcRSsiWk2fD3n1PmFVJEL11NXcKBGYcFbbtfwhIep7Z/7akDHgfX0YnekAyknHx2GxLh2gnNtDDH/BrifnL27ZZjqm9POS2Lnxg35gUhk1ixTn1wnC/S7wIatsQAmoPzW6fyJXNq8vyGvskTWO3jAXUssVZAPF2ZWarvKTD hughbroders@Hughs-SMU-MBP.hitronhub.home"
```
Comment (qawse3dr) [inline config.yaml]: You might want to remove this for security reasons if you can do an amend

File: purpl3_rms/src/components/computers/computers.js [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
             }
             }).then((res) => {
-              for(let entry of res.data.entries){
-                entry.name = entry.nickName;
-            }
-              setComputer_list(res.data.entries)
+                for(let entry of res.data.entries){
+                    entry.name = entry.nickName;
+                }
+                setComputer_list(res.data.entries)
```
Comment (qawse3dr) [inline purpl3_rms/src/components/computers/computers.js]: What's changed here just spacings?

**Review by qawse3dr — APPROVED** [review]
Just did a quick review looks good make sure mobile works as well before merging and remove your ssh key

File: purpl3_rms/src/components/computers/computers.js (hunk 2 above)
Comment (rachelbroders) [reply purpl3_rms/src/components/computers/computers.js]: yeah prob. I didnt like how some files had 2 spaces and others had 4 lol

**Pull request merged**